### ISSUE TITLE
[request] add easier build method

### ISSUE THREAD

**Issue by rockaut** [cavaliercoder/libzbxpython#6]
In times of docker an easy build method would be great ... ;-)

**Comment by cavaliercoder** [comment]
I agree. Unfortunately this module never really got much traction, so I've invested my limited energy elsewhere. If it's useful to you, PRs are certainly welcome!
